src/main/java/com/notably/model/suggestion/SuggestionModelImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.notably.model.suggestion;
+
+import java.util.List;
+import java.util.Optional;
+
+import javafx.beans.property.Property;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+/**
+ * The implementation class of SuggestionModel.
+ */
+public class SuggestionModelImpl implements SuggestionModel {
+    private ObservableList<SuggestionItem> suggestions;
+    private String commandInputText;
```

Comment:
Let's remove this and all the methods related to it. Keeping this here means we're duplicating the state of our application (since @HemanshuGandhi has already added a `CommandInputModel`)